src/registrar/models/utility/generic_helper.py

```diff
@@ -173,10 +172,6 @@ def _update_org_type_from_generic_org_and_election(self):
                 self.instance.is_election_board = None
             self.instance.organization_type = generic_org_type
         else:
-            # This can only happen with manual data tinkering, which causes these to be out of sync.
-            if self.instance.is_election_board is None:
-                self.instance.is_election_board = False
-
```

Comment:
This turns out not to be true. If you back out after only filling out the generic org form, is_election_board will be None.